content/5_Plat/Bin_Jump.mdx

```diff
@@ -215,9 +214,105 @@ export const problems = {
 	<Resource source="SecondThread" title="Tree Basics - Binary Lifting" url="https://codeforces.com/blog/entry/81527"> </Resource>
 </Resources>
 
+<Info>
+
+Binary jumping is more commonly referred to as "binary lifting."
+
+</Info>
+
 ### Solution
 
-<IncompleteSection />
+To solve this problem, we can use a standard binary lifting implementation where ```jmp(int x, int d)``` corresponds to the $d$th ancestor of $x$.
+
+In our ```jmp(int x, int d)``` if our final value of $x$ is $0$, then such a node does not exist and we can simply return $-1$. This is because the lowest number a node can be is $1$ (the root of the tree). 
+
+In our ```jmp(int x, int d)``` implementation, we can test if we jump in powers of two by using the $\&$ operator. If the $i$th bit on the right is toggled, then we jump. For example, a jump of $13$ would correspond to the binary number $1101$. We would jump 3 times on bits $$i$$.
+
+<Spoiler title="Illustration of the jump method">
+
+![Binary Lifting Diagram](./assets/binaryliftingjump.png)
+
+</Spoiler>
+
+To calculate the columns required for the ```up``` array, use the formula $\textrm{ceil}(\log_2{N})$ which in our case simplifies to $\textrm{ceil}(\log_2{(2\cdot 10^5)}) = 18$.
```

Comment:
use $\lceil \rceil$

Reply:
It's not clear why it's "columns" instead of "rows" here. Personally, I would say that `int up[MS][MX]; ` is an array of `MS` rows and `MX` columns.